docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -13,154 +13,220 @@ isTocNested: true
+Given the impact of distance on node-to-node communication, it is highly useful to be able to specify at a table level, how its data should be spread across the cluster. This way, we can easily move tables closer to their clients and decide which tables actually need to be geo-distributed. This can be achieved using YSQL Tablespaces. YSQL Tablespaces are entities that can specify the number of replicas for a set of tables/indexes, and how each of these replicas should be distributed across a set of cloud, regions, zones.
 
-In PostgreSQL, tablespaces are used for specifying a location on disk with options to control data access. That is, tablespaces group tables and indexes based on how you intend to store and access the data. For instance, you can choose to place heavily accessed smaller tables and indexes in SSD or “fast” storage compared to other tables.
+In this document, we will create a cluster that is spread across multiple regions across the world. We will then illustrate how to create tablespaces that specify single-zone, multi-zone and multi-region placement policies. We will then create tables/indexes associated with them and show the effect of geo-distribution on basic YSQL commands. This can be summarized in the below diagram.
 
-For YugabyteDB clusters, however, location does not pertain to disk locations. For a cloud-native distributed database, location pertains to the cloud, region, and zone where the data is supposed to be. Therefore, although YSQL tablespaces are built on PostgreSQL tablespaces that allow you to specify placement of data at a table level, this placement information in YSQL defines the number of replicas for a table and index, as well as how they can be distributed across a set of cloud, regions, and zones. 
+![Overview Cluster Diagram](/images/explore/tablespaces/overview_cluster_diagram.png)
 
-Note that you cannot use a tablespace outside of the cluster in which it is defined.
+The following table shows the results of the experiment being illustrated in the below section. It summarizes the effect of various geo-distribution policies on the latencies observed while running INSERTs and SELECTs.
 
-## Defining a Tablespace
+| Geo-Distribution | INSERT Latency (ms) | SELECT Latency (ms) |
+| :--------------- | :------------------ | :------------------ |
+| Single Zone | 4.676 | 1.880 |
+| Multi Zone | 11.825 | 4.145 |
+| Multi Region | 836.616 | 337.154 |
 
-You can define a tablespace using the following syntax:
+## Cluster Setup
+To illustrate the differences between single-zone, multi-zone and multi-region configuration, we will now deploy a cluster with the following topology (as shown in the above cluster diagrams). We choose this topology only for illustrative purposes as it can allow creation of node/zone/region fault tolerant placement policies in the same cluster with minimum nodes.
```

Comment:
The differences between single-zone, multi-zone and multi-region configuration becomes apparent when a cluster with the following topology (as per the preceding cluster diagrams) is deployed. This topology is chosen for illustrative purposes as it can allow creation of node, zone, region fault-tolerant placement policies in the same cluster with minimum nodes.